LICENSE.txt

```diff
@@ -25,3 +25,8 @@ INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
 CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 OF THE POSSIBILITY OF SUCH DAMAGE
+
+Copyright (c) 2003-2012 Christopher M. Kohlhoff (chris at kohlhoff dot com)
```

Comment:
I am thinking if this copyright line without much context can be misinterpreted. Perhaps adding this to the comment near the code from boost would make more sense?

Thinking a little more, I think all this can be moved to that file, and we only keep the copy of the license "outside". Does it make sense?

Reply:
Hmm, yea, just updated the request